test/org/opensolaris/opengrok/web/DirectoryListingTest.java

```diff
@@ -276,8 +275,8 @@ private void validateEntry(Element element) throws Exception {
         entry.lastModified = getLastModified(nl.item(3));
         try {
             entry.size = getSize(nl.item(4));
-        } catch (Exception e) {
-            entry.size = -1;
+        } catch (NumberFormatException e) {
```

Comment:
can this actually happen given that getSize() does not parse the integer anymore ?